framework/auth/cas.py

```diff
@@ -283,9 +283,10 @@ def make_response_from_ticket(ticket, service_url):
                 )))
 
             # if user is authenticated by CAS
+            # TODO [CAS-27]: Remove Access Token From Service Validation
             return authenticate(
                 user,
-                cas_resp.attributes['accessToken'],
+                cas_resp.attributes.get('accessToken', ''),
```

Comment:
Key `accessToken` may not exist. Added a TODO comment to remove releasing `accessToken` to OSF in the future.